app/src/main/java/com/veniosg/dir/android/adapter/FileHolderListAdapter.java

```diff
@@ -105,7 +105,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
 		holder.primaryInfo.setText(item.getName());
 		holder.secondaryInfo.setText(item.getFormattedModificationDate(convertView.getContext()));
 		// Hide directories' size as it's irrelevant if we can't recursively find it.
-		holder.tertiaryInfo.setText(item.getFile().isDirectory()? "" : item.getFormattedSize(
+		holder.tertiaryInfo.setText(item.getFile().isDirectory() ? item.getNumberOfItems(convertView.getContext()) : item.getFormattedSize(
```

Comment:
This should be extracted to `FileHolder#getSizeInfo(Context ctx)`

Reply:
Done!